cypress/support/pages/yves/company/company-role/index/repositories/suite-company-role-index-repository.ts

```diff
@@ -0,0 +1,7 @@
+import { injectable } from 'inversify';
+import { CompanyRoleIndexRepository } from '../company-role-index-repository';
+
+@injectable()
+export class SuiteCompanyRoleIndexRepository implements CompanyRoleIndexRepository {
+  getAddNewRoleButton = (): Cypress.Chainable => cy.get('a.button').contains('Add new role');
+}
```

Comment:
same here, avoid using text in your selectors. the better selector would be something like: a contains in href 'company-role/create'